doc/ref_model/chapters/chapter04.md

```diff
@@ -268,60 +268,162 @@ Table 4-10 shows performance measurement capabilities.
 
-**1)** These compute Flavours are intended to be used for transitional purposes and workload vendors are expected to consume smaller Flavours and adopt microservices-based designs for their workloads.<br>
-**2)** In Kubernetes based environments these are the resource requests of the containers in the pods. To get guaranteed resources the resource requests should be set to the same values as the resource limits, to get burstable resources the resource limits should be higher than the resource requests while to get best effort resources none of resource requests of resource limits should be set.<br>
-**3)** The "local disk" is an ephemeral disk that provides storage for the life of a VM/Pod, and can either be provided by storage devices housed within the physical server on which the VM/Pod is running, or from an external storage device such as SAN or NFS.
+<a name="4.2.1.2"></a>
+#### 4.2.1.2 High Performance Profile
+
+The high-performance profile (H) is intended to be used for workloads that require predictable performance, high network throughput requirements and/or low network latency. To satisfy predictable performance needs, NUMA alignment, CPU pinning, and Huge pages are enabled. For obvious reasons, the high-performance profile doesn’t support over-subscription.
 
 <a name="4.2.2"></a>
-### 4.2.2 Virtual Network Interface Specifications
+### 4.2.2 Profiles Specifications & Capability Mapping
+
+| Ref | Capability  | Basic | High Performance | Notes |
+|-----|---------|----------|----------|--------|
+| e.cap.006 | CPU pinning | No | Yes | Exposed performance capabilities as per Table 4-2 |
+| e.cap.007 | NUMA alignment  | No | Yes | |
+| e.cap.013 | SR-IOV over PCI-PT  | No   | Yes | |
+| i.cap.018 | Huge page support  | No  | Yes | Internal performance capabilities as per Table 4-7 |
+| | SMT | Yes | Yes | |
+| | Storage encryption | Yes | Yes | |
+| | DPDK | No | Yes| |
+| | CPU Architecture | \<value> | \<value> | Values such as x64, ARM, etc. |
+| | Host Operating System (OS) | \<value> | \<value> | Values such as specific Linux version, Windows |
+| | Hypervisor| \<value> | \<value> | Values such as KVM, Hyper-V, etc. when relevant, depending on technology |
+
+
+<a name="4.2.3"></a>
+### 4.2.3 Profile Extensions
+
+Profile Extensions represent small deviations from or further qualification of the profiles that do not require partitioning the infrastructure into separate pools, but that have specifications with a finer granularity of the profile. Profile Extensions provide workloads a more granular control over what infrastructure they can run on.
+
+| Profile Extension Name | Mnemonic | Applicable to Basic Profile | Applicable to High Performance Profile | Description | Notes |
```

Comment:
Profile extension should apply only to High Performance profiles

Reply:
@karinesevilla Could we, for example, host applications on edge sites for better user experience but there can be occasions where the experience may perform poorly?